spec/services/export_offenders_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe ExportOffenders do
+  describe '#call' do
+    let(:file) { Rails.root.join('tmp', 'soi_extract.csv') }
+
+    let!(:offender_1) do
+      create(:offender)
+    end
+
+    let!(:offender_2) do
+      create(:offender)
+    end
+
+    let!(:identity_1) do
+      create(:identity, offender: offender_1, noms_offender_id: '0001')
+    end
+
+    let!(:identity_2) do
+      create(:identity, offender: offender_1, noms_offender_id: '0002')
+    end
+
+    let!(:identity_3) do
+      create(:identity, offender: offender_2, noms_offender_id: '0003')
+    end
+
+    before { described_class.call(file) }
+
+    it 'exports offenders to csv file' do
+      lines = File.read(file).split("\n")
+
+      expect(lines[0]).to eq 'noms_offender_id,identity_id,offender_id'
```

Comment:
**[no blocker]** might simplify the display of the CSV expectation if you use [heredoc syntax](http://www.virtuouscode.com/2016/01/06/about-the-ruby-squiggly-heredoc-syntax/)